interface/usergroup/usergroup_admin.php

```diff
@@ -103,19 +103,6 @@
               $tqvar = formData('mname','P');
               sqlStatement("update users set mname='$tqvar' where id= ? ", array($_POST["id"]));
       }
-      if ($_POST["role_name"]) {
```

Comment:
This code needs to be left in for the full screen pr. You need to update your code and resubmit.

Reply:
At it.